etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -38,8 +39,7 @@
     public void executeRawLoad(String[] params) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "executeRawLoad" is 21 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)